Pax.sln

```diff
@@ -0,0 +1,59 @@
+﻿

```

Comment:
Nice, this patch seems to target this [issue](https://github.com/niksu/pax/issues/4)